sdk/src/workflow_context.rs

```diff
@@ -398,6 +398,14 @@ impl DrainableSignalStream {
         }
         signals
     }
+
+    pub fn drain_ready(&mut self) -> Vec<SignalData> {
```

Comment:
Intentionnal?

Reply:
Yeah, it's useful for something else I didn't include in the commit but is also useful generally, might as well throw it in